pkg/installation/url.go

```diff
@@ -35,7 +35,8 @@ func GetUrlType(u string) int {
 }
 
 func isK8sApiUrl(u string) bool {
-	return k8sApiURLRegexp.MatchString(u)
+	u = strings.SplitN(u, ":", 2)[0]
```

Comment:
This is removing the port? Might be more robust to use `url.Parse`.

Reply:
I feel like this is robust enough